toolchain/patches/edk2.patch

```diff
@@ -33,3 +33,37 @@ index f2bb624..8d03507 100755
  
  ###########################
  # CLANGPDB IA32 definitions
+diff --git a/BaseTools/Source/C/Makefiles/header.makefile b/BaseTools/Source/C/Makefiles/header.makefile
+index 0df728f..8eb9fb0 100644
+--- a/BaseTools/Source/C/Makefiles/header.makefile
++++ b/BaseTools/Source/C/Makefiles/header.makefile
+@@ -92,14 +92,14 @@ BUILD_CFLAGS = -MD -fshort-wchar -fno-strict-aliasing -fwrapv \
+ -Wno-unused-result -nostdlib -g
+ else
+ BUILD_CFLAGS = -MD -fshort-wchar -fno-strict-aliasing -fwrapv \
+--fno-delete-null-pointer-checks -Wall -Werror \
++-fno-delete-null-pointer-checks -Wall -Werror -Wno-error=use-after-free -Wno-error=dangling-pointer= \

```

Comment:
The -Wno-error=dangling-pointer= was added because of a warning (-Werror) about assignment of a pointer to a stack-allocated variable in edk2.